Gordon360/Services/ProfileService.cs

```diff
@@ -235,18 +237,18 @@ public async Task UpdateProfileImageAsync(string username, string? path, string?
 
 
         /// <summary>
-        /// Sets the path for the profile links.
+        /// Sets the component of the Custom_profile.
         /// </summary>
         /// <param name="username">The username</param>
         /// <param name="type"></param>
         /// <param name="links"></param>
-        public async Task UpdateProfileLinkAsync(string username, string type, CUSTOM_PROFILE links)
+        public async Task UpdateCustomProfileAsync(string username, string type, CUSTOM_PROFILE links)
```

Comment:
```suggestion
        public async Task UpdateCustomProfileAsync(string username, string type, CUSTOM_PROFILE (othername))
```

links is not the correct variable name since we now also hold PlannedGradYear